copyright_and_licenses.csv

```diff
@@ -195,7 +195,8 @@ nl_nsgi_rdcorr2018.tif,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI
 nl_nsgi_rdtrans2018.tif,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI),CC-BY-4.0,,
 nl_nsgi_README.txt,Disclaimed,Public domain,,
 no_kv_arcgp-2006-sk.tif,Kartverket (SK),CC-BY-4.0,1.11,
-no_kv_CD_above_Ell_ETRS89_v2021a.tif,Kartverket (SK),CC-BY-4.0,1.9,
+no_kv_CD_above_Ell_ETRS89_v2021a.tif,Kartverket (SK),CC-BY-4.0,,1.9
```

Comment:
Here is modified, but the rest of the PR is deleting the file. That is not consistent.